.github/workflows/main.yml

```diff
@@ -20,33 +20,56 @@ jobs:
 
       - name: Lint Ansible roles/servicetelemetry/ directory
         run: ${HOME}/.local/bin/ansible-lint roles/servicetelemetry
-# TODO: requires a bunch of work on our bash scripts, or finesse
-#      - name: Run Super-Linter
-#        uses: github/super-linter@v3
-#        env:
-#          DEFAULT_BRANCH: master
-#          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-  build-check:
-    name: Build check
+
+  build-operator-check:
+    name: Build Operator check
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Verify image builds
+        run: docker build --tag infrawatch/service-telemetry-operator:latest --file build/Dockerfile .
```

Comment:
NIT This is for my learning mostly... shouldn't we prioritize using podman instead of docker? Not sure if the ubuntu-20.04 image that is being used as base includes the required repos

Reply:
For the purposes of what we're doing here, I'm not sure there is a huge difference. We're simply checking that a build is possible. I agree we should use podman, but I'm not sure what version is on ubuntu-20.04, what repos are necessary, and I'd prefer not to debug problems with podman here. I'm going to defer on this one for now.

Reply:
+1, thanks